### PR TITLE
Item repo methods

### PR DESCRIPTION
**Pull request by jrwhitmer**
Hey y'all!

In this PR we created and tested all of the methods required for our item_repo class. We also included some helper methods for our update method. Possible refactoring might be needed for our create method.

We also expanded our testing on our items and merchant classes within this PR.

### CODE REVIEW

File: lib/items.rb [hunk 1]
```diff
@@ -1,20 +1,35 @@
...
-    @merchant_id = attributes[:merchant_id]
+    @updated_at = attributes[:updated_at]
+    @merchant_id = attributes[:merchant_id].to_i
+  end
+
+  def unit_price_to_dollars
+    @unit_price.to_f
+  end
+
+# talk to instructor about using these helper methods
```
[the remark this answers is not in the document]
Comment (ahenzler) [reply lib/items.rb:27]: we should need to go through and remove whitespace

File: spec/items_repo_spec.rb [hunk 2]
```diff
@@ -1,12 +1,161 @@
...
 require './lib/sales_engine'
 require './lib/items'
 require './lib/items_repo'
 require './lib/merchants'
 require './lib/merchants_repo'
 
+
 RSpec.describe ItemRepo do
 
+  se = SalesEngine.from_csv({
```
Comment (deebot10) [inline spec/items_repo_spec.rb:12]: All test look great, with the refactor. Think we are in a great spot to move on to our next iteration

**Review by deebot10 — APPROVED** [review]

File: lib/items.rb [hunk 3]
```diff
@@ -1,20 +1,35 @@
...
+                :updated_at,
+                :merchant_id
 
   def initialize(attributes)
-    @id = attributes[:id]
+    @id = attributes[:id].to_i
     @name = attributes[:name]
     @description = attributes[:description]
-    @unit_price = attributes[:unit_price]
+    @unit_price = (BigDecimal(attributes[:unit_price])/100)
```
[the remark this answers is not in the document]
Comment (rcasias) [reply lib/items.rb:17]: I think we can actually remove the parentheses here, It might look a bit cleaner.

File: spec/merchants_repo_spec.rb [hunk 4]
```diff
@@ -41,44 +41,41 @@
...
 
       mr.update(12337412, attributes)
 
       expected = mr.find_by_id(12337412)
-      expect(expected.name).to eq "TSSD"
+      expect(expected.name).to eq("NewStore67")
 
-      expected = mr.find_by_name("Turing School of Software and Design")
-      expect(expected).to eq nil
+      expected = mr.find_by_name("Regina is cool")
```
Comment (ahenzler) [inline spec/merchants_repo_spec.rb:71]: love this!!!

**Review by ahenzler — APPROVED** [review]

File: lib/items.rb (hunk 3 above)
Comment (rcasias) [inline lib/items.rb:17]: I think we can actually remove the parentheses here, It might look a bit cleaner.

File: lib/items.rb [hunk 5]
```diff
@@ -1,20 +1,35 @@
...
+    @unit_price.to_f
+  end
+
+# talk to instructor about using these helper methods 
+  def created_at_to_time
+    Time.parse(@created_at)
+  end
+
+  def updated_at_to_time
+    Time.parse(@updated_at)
```
Comment (rcasias) [inline lib/items.rb:33]: We might need this method to stay within the created_at and Updated_at in initialize.

**Review by rcasias — COMMENTED** [review]
Everything looks good, just a few things to look into and ask about, but i think this is ready to merge and move to iteration 1.

**Pull request merged**